hazelcast/src/main/java/com/hazelcast/jet/config/JetConfig.java

```diff
@@ -29,6 +29,8 @@
 
     private InstanceConfig instanceConfig = new InstanceConfig();
     private EdgeConfig defaultEdgeConfig = new EdgeConfig();
+    private boolean enabled = true;
+    private boolean uploadResources;
```

Comment:
Did you consider reusing user code deployment property? What were the arguments against?

Also, I don't particularly like the name, maybe `resourceUploadEnabled`?

Reply:
I don't think we should use user code deployment(UCD) property here, they are not totally same. for UCD, if you upload something it lives there forever and it is usable from anywhere, entry-processor, executor-service... With this flag we are letting the user to upload resources which will only be used in the submitted job.

`resourceUploadEnabled` sounds good 👍